msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java

```diff
@@ -1794,6 +1794,24 @@ private boolean isAllowToFieldMyGroupMembers(User user)
     else
       return false;
   }
+  
+  @Override 
+  public boolean isAllowToFieldMyGroupRoles() {
+	 LOG.debug("isAllowToFieldMyGroupRoles()");
```

Comment:
logging should be more descriptive and the method name is automatically output so no need to for it.